src/olympia/reviewers/tests/test_utils.py

```diff
@@ -760,18 +760,70 @@ def test_public_addon_confirm_auto_approval(self):
     def test_public_with_unreviewed_version_addon_confirm_auto_approval(self):
         self.grant_permission(self.request.user, 'Addons:PostReview')
         self.setup_data(amo.STATUS_PUBLIC, file_status=amo.STATUS_PUBLIC)
-        AutoApprovalSummary.objects.create(
-            version=self.version, verdict=amo.AUTO_APPROVED)
+        self.current_version = self.version
+        summary = AutoApprovalSummary.objects.create(
+            version=self.version, verdict=amo.AUTO_APPROVED, weight=152)
         self.version = version_factory(
             addon=self.addon, version='3.0',
             file_kw={'status': amo.STATUS_AWAITING_REVIEW})
         self.file = self.version.files.all()[0]
         self.helper = self.get_helper()  # To make it pick up the new version.
         self.helper.set_data(self.get_data())
 
-        # Confirm approval action should not be available since the latest
-        # version is not public (#5604)
```

Comment:
This is no longer true since the last comments in #6894 are asking to change this.